devlab/utils/README.md

```diff
@@ -0,0 +1,16 @@
+1. Deploy development lab with changed IP addresses(standard IPs like 192.168.1. will be changed to your current IPs from devlab/config.ini)
+    cd CloudFerry
+    change IPs in devlab/config.ini
+    bash devlab/utils/deploy_lab.sh
+
+2. Clean environment(be carefull, all VMs, images, volumes, networks, flavors should be deleted)
+    cd CloudFerry
+    change IPs in devlab/config.ini
+    bash devlab/utils/os_cli.sh clean src
+    and/or
+    bash devlab/utils/os_cli.sh clean dst
+
+3. Clean environment and run migration
+    cd CloudFerry
+    change IPs in devlab/config.ini
+    bash devlab/utils/clean_run.sh
```

Comment:
Readme does not seem very useful, because each script already has documentation in the header. Looks like unnecessary double work in the future to support both.

If you believe we need it:
1. Please use markdown syntax for code (```)
2. Let's limit line length to 120 symbols
3. There's also `repackage_box.sh` script in this folder let's have description for it in README too.